ci/README.md

```diff
@@ -1,4 +1,5 @@
 # Continuous Integration
```

Comment:
This folder / file should not exist anymore. (was deleted in Brent's CI related PRs). Be sure to resolve the merge conflict in favor of it being deleted since all CI scripts are now in the github workflows folder.

Reply:
Why did this file get created? It looks like it was autogenerated by Actions in 24f2ad76c81b31072a5100b3049cfc2e66f87ada

Reply:
I am unsure where this came from? It looks like it was generated in 24f2ad76c81b31072a5100b3049cfc2e66f87ada.